dovecot/edit_net.cgi

```diff
@@ -34,19 +34,24 @@ print &ui_table_row($text{'net_ssl_disable'},
 
 @listens = &find("imap_listen", $conf, 2) ?
 		("imap_listen", "pop3_listen", "imaps_listen", "pop3s_listen") :
-		("listen", "ssl_listen");
+		("listen");
 foreach $l (@listens) {
 	$listen = &find_value($l, $conf);
 	$mode = !$listen ? 0 :
-	        $listen eq "[::]" ? 1 :
+		# All interfaces, put in any order, e.g. "[::], *" or "*, ::"
+	        $listen =~ /^(\*|::|\[::\]),\s*(\*|::|\[::\])$/ ? 1 :
+		# IPv6 only, e.g. "[::]" or "::"
+	        $listen =~ /^(?:\:\:|\[\:\:\])$/ ? 4 :
```

Comment:
Just use `eq` here instead of a regexp

Reply:
Good suggestion, thanks! Fixed in the latest commit!